src/DvfHelpers.php

```diff
@@ -7,7 +7,7 @@
 use Drupal\Component\Render\FormattableMarkup;
 
 /**
- * Class DvfHelpers.
+ * Provide functions for help pages.
```

Comment:
This is an incorrect description, but its fine :)

Reply:
good catch, will fix since making changes now